File: 2.Warmup/2.3. Loops 01/basics.js
// Print numbers from 0 to 10
console.log("-----------------------")
for (let i = 0;i<=10;i++){
    console.log("Hello World " + i)
}

// Print even numbers from 0-10
console.log("-----------------------")
for(let i = 0;i<=10;i =i+2){
    console.log("Hello World " + i)
}

// Print numbers from 10-1
console.log("-----------------------")
for(let i = 10;i>0;i--){
    console.log("Hello World " + i)
}

// Print the items in the array
console.log("----------- Print the items in the array --------")
const arr = [1,2,3,4,5,6,7,8]
for(let i = 0;i < arr.length ;i++){
    console.log(arr[i])
}

// Print only even items in the array
console.log("----------- Print only even items in the array --------")
for(let i = 0;i < arr.length ;i++){
    if(arr[i] %2 === 0){
        console.log(arr[i])
    }
}

// Print numbers from 0 to 10 using while
console.log("--------- Print numbers from 0 to 10 using while --------------")
let i = 0
while(i <=10){
    console.log(i)
    i++
}

// Print numbers from 10-1 using while
console.log("---------- Print numbers from 10-1 using while -------------")
let j = 10
while(j>0){
    console.log(j)
    j--
}

File: 2.Warmup/2.3. Loops 01/basics.py
# Print numbers from 0 to 10 exculding 10
print("---------- Print numbers from 0 to 10 exculding 10 -------------")
for i in range(0,11):
    print(f"Hello World {i}")

# Print even numbers from 0-10
print("--------- Print even numbers from 0-10 --------------")
for i in range(0,11,2):
    print(f"Hello World {i}")

#Print numbers from 10-1
print("---------- Print numbers from 10-1 -------------")
for i in range(10,0,-1):
    print(f"Hello World {i}")

# Print the items in the array
print("----------- Print the items in the array --------")
arr = [1,2,3,4,5,6,7,8,9]
for i in range(0,len(arr)):
    print(i)

# Print only even items in the array
print("----------- Print only even items in the array --------")
for i in range(0,len(arr)):
    if arr[i] %2 == 0:
        print(arr[i])

# Print numbers from 0 to 10 exculding 10 using while loop
print("---------- Print numbers from 0 to 10 exculding 10 using while loop-------------")
i = 0
while i<11:
    print(i)
    i = i + 1

# Print even numbers from 0-10 using while loop
print("--------- Print numbers from 10-1 using while loop --------------")
j = 10
while j > 0:
    print(j)
    j = j-1